### PR TITLE
FilterBlocks in InterpretationTab, Version 1

### PR DESCRIPTION
**Pull request by alexkb0009**
Should work as could be expected/intuited.

[Video Preview](https://gyazo.com/7005e6af8a0dd8718dcc363293f139c8)

Screenshot:

### CODE REVIEW

File: src/encoded/static/components/item-pages/CaseView/VariantSampleListController.js [hunk 1]
```diff
@@ -150,6 +150,51 @@ export class VariantSampleListController extends React.PureComponent {
         // Using embed API instead of datastore=database in order to prevent gene-list related slowdown
         this.setState({ "isLoadingVariantSampleListItem": true }, () => {
 
+
+            function commonVSEmbeds(prefix){
+                return [
```
Comment (Bianca-Morris) [inline src/encoded/static/components/item-pages/CaseView/VariantSampleListController.js:155]: Why did you move this into `VariantSampleListController`? Doesn't seem like these are using state or anything, and now they're being re-created whenever `fetchVariantSampleItem` is called (not often necessarily, but more often than they would have to be otherwise).

**Review by Bianca-Morris — APPROVED** [review]
Looks like the issues I ran into earlier have been resolved. Works nicely. I had a couple of small notes/questions, but nothing that's really urgent. I'd also maybe throw the link into the `#cgap-frontend` channel on slack, just to get some other people to test it, too, on the off chance I missed anything.

File: src/encoded/static/components/browse/variantSampleColumnExtensionMap.js [hunk 2]
```diff
@@ -337,7 +338,7 @@ export const StructuralVariantTranscriptColumn = React.memo(function StructuralV
 });
 
 export const ProbandGenotypeLabelColumn = React.memo(function ProbandGenotypeLabelColumn(props){
-    const { result, align = "center", showTips = true, truncate = true, showIcon = true } = props;
+    const { result, align = "left", showTips = true, truncate = true, showIcon = true } = props;
```
[the remark this answers is not in the document]
Comment (alexkb0009) [reply src/encoded/static/components/browse/variantSampleColumnExtensionMap.js:341]: Shannon and I figured it'd look better w. icons aligned to each other on the left; in cases of longer genotypes the text would appear to have same width due to truncation on some lines, [the icons appear misaligned](https://gyazo.com/0771d654f76690ae6d345b172b84b2e9) like a bug/glitch.

File: src/encoded/static/components/item-pages/CaseView/VariantSampleListController.js (hunk 1 above)
Comment (alexkb0009) [reply src/encoded/static/components/item-pages/CaseView/VariantSampleListController.js:155]: Good call, changed.

File: src/encoded/static/components/browse/variantSampleColumnExtensionMap.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexkb0009) [reply src/encoded/static/components/browse/variantSampleColumnExtensionMap.js:341]: Could potentially do something like [this](https://gyazo.com/7beced83f293dc19a3b191a57a0ce340) but not 100% sure if is better (looks like [this](https://gyazo.com/db6d627bfa48b1b01ed2d2eeb07169a8) when keep expanding) + don't want to spend too much time implementing atm.

File: src/encoded/static/components/item-pages/CaseView/VariantSampleListController.js (hunk 1 above)
Comment (alexkb0009) [reply src/encoded/static/components/item-pages/CaseView/VariantSampleListController.js]: OTOH tangentially, keeping a constant global object with fields instead of (re)generating it on the fly may theoretically incur more memory usage ... but it'd be very insignificant especially considering entirety of downloaded code is stored in browser memory anyway (including function body that generates it).
